scenes/ui/UI.tscn

```diff
@@ -0,0 +1,66 @@
+<<<<<<< HEAD
```

Comment:
Merge Conflict boy, please resolve
contact if you need help

Reply:
Oh yeah, you are right. Miss that one. No notification from IDE